### PR TITLE
[Enhancement](jdbc catalog) Add a property to test the connection when creating a Jdbc catalog

### PR DESCRIPTION
**Pull request by zy-kkk**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

File: gensrc/proto/internal_service.proto [hunk 1]
```diff
@@ -662,6 +662,17 @@ message PFetchTableSchemaResult {
   repeated PTypeDesc column_types = 4;
 }
 
+message PJdbcTestConnectionRequest {
+    required bytes jdbc_table = 1;
```
Comment (morningman) [inline gensrc/proto/internal_service.proto:666]: use optional

**Comment by zy-kkk** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run feut

**Review by AshinGau — APPROVED** [review]

**Pull request merged**